### DISCUSSION TITLE
Dark/Light mode toggle only showing after deployment

### DISCUSSION THREAD

**Discussion by gael-vanderlee** [alshedivat/al-folio#2356] [Q&A] [answered]
Hello, 
thanks for the awesome template. I've noticed that there are some slight differences between my local (docker) site and the deployed (github pages) site.
Notably, the dark mode toggle in the nav bar only appears when the site is deployed. Any idea where I should start looking to understand the root problem on my local website ?

**Comment by george-gca** [comment]
This is weird. Currently we use only for this toggle [tabler icons](https://tabler.io/icons), introduced in #2261. Probably for some reason you are being unable to use them locally. Maybe check your [Chrome](https://developer.chrome.com/docs/devtools/overview/) or [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/) dev tools, in the `console` tab, to check for errors there.

**Comment by gael-vanderlee** [comment, marked as answer]
Very strange issue, investigating I found a couple more things:
- No messages in the console
- All icons were missing, socials at the bottom of the landing page, cv icons etc
- The issue was on one machine only, another windows machine worked eventhough the repos were synched 
- The icons worked when I pulled the original al-folio.

I tried removing the folder that weren't added to git (_site or .jekyll_cache) since they're the only ones that could have differed between the machines. Still no icons.
Ended up **deleting the whole project locally and re-cloning, that fixed it**. 
Still not sure about the source of the issue, but hopefully my answer can help someone in the future if it happens to someone else.
